### PR TITLE
[Docs](mlu-ops): Add logspace/lgamma in rst and op_list.

### PR DESCRIPTION
**Pull request by mahxn0**
Thanks for your contribution and we appreciate it a lot. :rocket::rocket:

## 1. Motivation
Add logspace/lgamma in rst and op_list.

## 2. Modification
docs/MLU-OPS-OpList.md
docs/design_docs/lgamma/lgamma.md
docs/user_guide/9_operators/index.rst

### CODE REVIEW

**Review by DanieeelLiu — APPROVED** [review]
LGTM

File: docs/user_guide/9_operators/index.rst [hunk 1]
```diff
@@ -757,3 +757,39 @@ mluOpExecFFT
 - ``y`` 为输出信号。
 - :math:`DFT_{N}` 为长度为N傅里叶变换的变换矩阵。
 
+.. _logspace:
+
+mluOpLogspace
+---------------
+该算子输出长度为`steps`的数组，其值为以base为底数，以[start,end]区间上均分部分的数为指数生成的幂。
```
Comment (Unireverse) [inline docs/user_guide/9_operators/index.rst:764]:
```suggestion
该算子输出长度为 `steps` 的数组，其值为以base为底数，以[start,end]区间上均分部分的数为指数生成的幂。
```

Comment (Unireverse) [reply docs/user_guide/9_operators/index.rst:764]: 这里也可以把外面包的符号去掉，或者使用``xxxx``

**Review by chqy99 — APPROVED** [review]

**Comment by chqy99** [comment]
LGTM

File: docs/user_guide/9_operators/index.rst (hunk 1 above)
Comment (mahxn0) [reply docs/user_guide/9_operators/index.rst]: 已修改

**Review by Unireverse — APPROVED** [review]

**Pull request merged**